datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +200,115 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self) -> Iterable[Tuple[str, str]]:
+
+        url = self.config.get_sql_alchemy_url()
+        engine = create_engine(url, **self.config.options)
+
+        with engine.connect() as conn:
+            inspector = inspect(conn)
+            schema = inspector.get_schema_names()  # returns a list
+            views = inspector.get_view_names(
+                schema[0])  # returns a list
```

Comment:
this indenting looks wrong, make sure to run `task lint` and `task fix` before opening the PRs